windows/src/support/texteditor/FontSelect.cpp

```diff
@@ -0,0 +1,116 @@
+/*
+ * TextEditor is copyright (C) SIL International. MIT License.
+ * Implementation of the FontSelect Class
+ */
+
+#include "FontSelect.h"
+#include "commdlg.h"
+#include "framework.h"
+
+font_select::font_select() {
+  default_font();
+}
+
+font_select::font_select(HFONT current_font) {
```

Comment:
When passing in a font handle in the constructor, we should clone the LOGFONT data that it references (using `GetObject`) so that we own the handle, unless it is explicitly documented that passing the handle in also gives ownership of the handle.

Reply:
Ok will change this